src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingRequirementType.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum HearingRequirementType {
+
+    MUSTINC("mustinc", "MUSTINC", "Must Include"),
+    EXCLUDE("exclude", "EXCLUDE", "Must Exclude"),
+    OPTINC("optinc", "OPTINC", "Optional Include");
+
+    private final String type;
+    private final String key;
+    private final String description;
+
+    @JsonValue

```

Comment:
Following our call earlier I think we should keep the actual Enum values to make serialisation/deserialisation easier